packages/apollo-fragment-react/package.json

```
@@ -42,10 +42,10 @@
     }
   ],
   "peerDependencies": {
-    "apollo-link-state": "0.4.1",
-    "react-apollo": "^3.0.0-beta.3"
+    "@apollo/client": "^3.2.0"
```

```suggestion
    "@apollo/client": "^3.5.6"
```

At this point, `@apollo/client` is up to 3.5.6